Makefile

```diff
@@ -12,6 +12,7 @@ DEIS_PROFILE ?= usw
 DEIS_APP ?= kumascript-dev
 PRIVATE_IMAGE ?= ${PRIVATE_REGISTRY}/${DEIS_APP}\:${VERSION}
 TEST_RUN_ARGS ?=
+TEST_RUN_TIMEOUT ?= 10000
```

Comment:
This worries me a little.  Are there tests that are making real network calls to MDN or GitHub?  It makes me think we should auto-stub some of the API calls to fail if they are called "for real" rather than with stub data.  We can re-evaluate if it becomes a problem.